src/operator/api/v1alpha3/clientintents_types.go

```diff
@@ -323,6 +323,10 @@ func (in *Intent) GetTargetServerName() string {
 		return OtterizeInternetTargetName
 	}
 
+	if in.Type == IntentTypeAWS || in.Type == IntentTypeGCP || in.Type == IntentTypeAzure || in.Type == IntentTypeDatabase {
```

Comment:
👍 
FYI: Once @omris94's changes go in, we should be able to replace this with a generic condition (either in.Kind == "", or if we end up settling on a schema that has a different way to differentiate internal from external targets)